src/Sdk/DTObjectTemplating/ObjectTemplating/Schema/MappingDefinition.cs

```diff
@@ -30,8 +30,7 @@ internal MappingDefinition(MappingToken definition)
                                     foreach (var propertiesPair in properties)
```

Comment:
Copied from SDK, not related for outputs but required for support 
```json
foo: 
{
   "requires": "true",
   "type": "runs-on",
   
}
```
in workflow schema.